main/src/com/google/refine/model/Cell.java

```diff
@@ -60,7 +60,7 @@
     @JsonIgnore
     final public Serializable   value;
     @JsonIgnore
-    final public Recon          recon;
```

Comment:
This spacing is intentional I think (so that all the field names are aligned as a column).